docs/pages/versions/unversioned/guides/customizing-webpack.md

```diff
@@ -2,28 +2,55 @@
 title: Customizing Webpack
 ---
 
-Expo uses the `@expo/webpack-config` to cover the majority of use-cases involved with running a react-native project in the browser. This is akin to `react-scripts` with `create-react-app`.
+When you run `expo start --web` or `expo build:web` the CLI will check to see if your project has a `webpack.config.js` in the root directory. If the project doesn't then Expo will continue it's process with the default `@expo/webpack-config` (preferred).
 
-You will more than likely need to customize your webpack config in a large scale or production application. The best way to do this is to extend or rebuild the webpack config using methods exported from `@expo/webpack-config`.
+> This is akin to `react-scripts` & `create-react-app`.
 
-By default `expo-cli` checks to see if a `webpack.config.js` files exists in the root directory before using `@expo/webpack-config`. This should provide you with a simple interface for creating complex configurations.
+If you need to edit the config the best way to do this is by running `expo customize:web` and selecting the `webpack.config.js` option.
+This will install `@expo/webpack-config` as a devDependency and create a template `webpack.config.js` into your project.
+You can now make changes to the default config then return it for the CLI to use.
+Deleting the config will cause Expo to fallback on the default again.
 
-## Extending the Default Config
+If you create a new Webpack config or make any changes to it you'll need to restart your Webpack dev server with `expo start --web`.
 
-Say you want to add a file type that isn't supported by default.
-Simply create a `webpack.config.js` in your project root and merge in a new loader.
+## Example
 
 **`webpack.config.js`**
 
 ```ts
-const merge = require('webpack-merge');
-// This will automatically get the dev/prod config based on process.env.NODE_ENV.
-const expoConfig = require('@expo/webpack-config');
-
-// Expo expects a function so we can pass around options.
-module.exports = function(env, argv) {
-  return merge(expoConfig(env, argv), {
-    // It's clobbering time...
-  });
+const createExpoWebpackConfigAsync = require('@expo/webpack-config');
+
+// Expo CLI will await this method so let's future proof
```

Comment:
You don't need async for future-proofing -- Expo CLI won't crash if it's not async. Just need to mention that async functions are supported as well.